modules/server/src/esClient/index.ts

```diff
@@ -0,0 +1,27 @@
+import { Client, type ClientOptions } from '@elastic/elasticsearch';
+
+export const buildEsClient = (esHost = '', esUser = '', esPass = '') => {
+	if (!esHost) {
```

Comment:
wouldn't it be fair to provide a default value for this?
ES has a common 9200 port, and localhost is a usual